packages/engine-server/src/workspace/service.ts

```diff
@@ -874,9 +879,8 @@ export class WorkspaceService implements Disposable, IWorkspaceService {
   }
 
   createConfig() {
-    // This line actually does something: it will create a config if one doesn't exist.
     // eslint-disable-next-line no-unused-expressions
```

Comment:
nit: in this case, we can also remove the eslint line right?

Reply:
you're right. fixed!